src/film/movieList/redux/MovieListSlice.js

```diff
@@ -0,0 +1,175 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import {
+  getPostsFromServer,
+  getPostsFromServerFilter,
+  getPostsFromServerSort,
+  getPostsFromServerSearch,
+  getPostFromServerById,
+  deletePostsFromServer,
+  editPostFromServer,
+  addPostFromServer,
+} from "./GetMovieListAPI";
+
+const initialState = {
+  posts: [],
+  loading: true,
+  currentFilter: "",
+  currentSort: "",
+  currentCountMovie: 0,
```

Comment:
Can we get use `posts.length` instead of `currentCountMovie`? It is nice to have only one source of truth in state.